### ISSUE TITLE
[FEATURE REQUEST] Display incoming trunk commits

### ISSUE THREAD

**Issue by dodalovic** [zielu/GitToolBox#193]
First of all - thanks for your wonderful product!

Currently, we have 2 options, either to display incoming / outgoing commits by specifying **Parent branch mode**, which works just fine. 

Personally, I feel it would be awesome if I can leave this option to **Auto** as is, and be able to have a new config option to somehow display that my feature branch **is running behind** specified `trunk` (e.g. `origin/master`) and display it somehow slightly differently (more transparent) than incoming changes from remote tracked branch. 

That way, I could both track remote tracking branch changes, as well as how much I'm running behind trunk.

Many thanks!

**Comment by zielu** [comment]
So with **Auto** it should display 3 things for feature branch (`FB`) pulled out from `master`:
- how much `master` is behind `origin/master`
- how much `FB` is behind `origin/FB`
- how much 'FB' diverged from `origin/master`

**Comment by dodalovic** [comment]
@zielu Having these 3 things shown in some intelligent way would be impressive. Are you proposing this or you're saying that this can be displayed somehow with the current plugin?

**Comment by zielu** [comment]
Let me rephrase, I haven't noticed that comment was saved :) 

For feature branch (`FB`) pulled out from `master`:

Currently **Auto** displays how much `FB` has diverged from `origin/master`:
- how many commits `origin/master` has are not in `FB` (incoming commits)
- how many commits `FB` has that are not in `origin/master` (outgoing commits)
This depends however on Task Management - you have to create feature branches using tasks.

What you would like to have is something like this ?:
- how much `master` is behind `origin/master`
- how much `FB` is behind `origin/FB`
- how much 'FB' diverged from `origin/master` (commits not present in one side or the other)

**Comment by dodalovic** [comment]
Yes, that'd be awesome! That'd help me get all the info needed. 
I'm not sure, though, if this is only me finding that valuable.
